README.md

```diff
@@ -20,20 +20,16 @@ You can use this library either as a Node.js module or inside an HTML page.
     ```
     npm install --save ion-js
     ```
-1. Use the library to read/write ion data. Here is an example that reads Ion data from a Javascript string
+1. Use the library to read/write ion data. Here is an example that reads Ion data from a Javascript string:
     ```javascript
-    var ionJs = require("ion-js")
-
-    var ionData = "{ hello: \"Ion\" }";
-    var ionReader = ionJs.makeReader(ionData);
-    ionReader.next();
-    ionReader.stepIn();
-    ionReader.next();
-    var hello = ionReader.fieldName();
-    var ion = ionReader.stringValue();
-    ionReader.stepOut();
-    console.log(ion.concat(", ").concat(hello));
+    let ion = require("ion-js");
+   
+    let ionData = '{ greeting: "Hello", name: "Ion" }';
+    let value = ion.load(ionData);
+    console.log(value.greeting + ", " + value.name + "!");
     ```
+   
```

Comment:
it would be nice to have an example of using one of the `dump*()` methods, too

Reply:
Added.